feature-pack/src/main/resources/modules/system/layers/base/org/jboss/resteasy/resteasy-jaxrs/main/module.xml

```diff
@@ -46,5 +65,7 @@
         <module name="javax.servlet.api"/>
         <module name="org.jboss.resteasy.resteasy-validator-provider-11" optional="true" services="export" export="true"/>
         <module name="org.jboss.logging"/>
+        <module name="org.reactivestreams" export="true"/>
+        <module name="org.eclipse.microprofile.restclient" export="true"/>
```

Comment:
org.reactivestreams and org.eclipse.microprofile.restclient are public modules

 * is it OK to expose all all their classes ?
 * there is no analysis for these thing, is it going to be unsupported for now ?
I can imagine both can be appreciated in micro-service world / wf-swarm.

 * if they are unsupported, can be the modules switched off and enabled on request - e.g. system property?
we need probably think about such functionality so jboss-modules support `<conditions>` for `module` elements similar to `<artifact`.  @kabir is there plan for such thing ? There should be :) We need to be able to hide some stuff which is still under development, but a) is tight to something else and the relevant part is done b) we need feedback from early adopters of such feature

Reply:
I definitely want to provide the analysis for those 2 things, just give me some time.
In the mean time, we can declare the module as privates if that really concerns you, or we can simply state that the features are tech preview in WFLY 12.

Reply:
+1 for private module